cardano_node_tests/tests/conftest.py

```diff
@@ -236,17 +236,18 @@ def session_autouse(change_dir: Any, close_dbconn: Any, testenv_setup_teardown:
 
 
 @pytest.fixture(scope="module")
-def testfile_temp_dir(tmp_path_factory: TempdirFactory, this_testfile: str) -> Path:
+def testfile_temp_dir(tmp_path_factory: TempdirFactory) -> Path:
     """Return a temporary dir for storing test artifacts.
 
     The dir is specific to a single test file.
     """
-    p = (
-        temptools.get_pytest_worker_tmp(tmp_path_factory)
-        .joinpath(helpers.get_id_for_mktemp(this_testfile))
```

Comment:
This was the only use of `helpers.get_id_for_mktemp` function, so it can be removed from `helpers` module now.